addons/hr/models/hr_department.py

```diff
@@ -47,11 +47,32 @@ def _search_has_read_access(self, operator, value):
         if operator not in supported_operators or not isinstance(value, bool):
             raise NotImplementedError()
         if not value:
-            return [(1, "=", 0)]
-        if self.env['hr.employee'].has_access('read'):
-            return [(1, "=", 1)]
-        departments_ids = self.env['hr.department'].sudo().search([('manager_id', 'in', self.env.user.employee_ids.ids)]).ids
-        return [('id', 'child_of', departments_ids)]
+            raise ValueError()
+        if self.env.user.has_group('hr.group_hr_user'):
+            return [(1, '=', 1)]
+        departments_ids = self.env['hr.department'].sudo()._search([
+            ('manager_id', 'child_of', self.env.user.employee_ids.ids)]
+        )
+        return [
+            ('id', 'child_of', departments_ids)
+        ]
+
+    @api.depends_context('uid', 'company')
+    @api.depends('manager_id')
+    def _compute_has_read_access(self):
+        if self.env.user.has_group('hr.group_hr_user'):
+            for r in self:
+                r.has_read_access = True
```

Comment:
```suggestion
            self.has_read_access = True
```
I think this will work :)